### PR TITLE
Changed the Kubernetes Container Runtime to containerd

### PR DESCRIPTION
**Pull request by danielapetito**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
Change the Kubernetes Container Runtime to containerd

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:
dojot/dojot#1918

### CODE REVIEW

File: roles/k8s-cluster/all-nodes/tasks/main.yaml [hunk 1]
```diff
@@ -17,51 +15,69 @@
     - gnupg2
     - software-properties-common
 
-- name: Add an apt signing key for Docker
-  tags:
-  - docker
+- name: Add an apt signing key for containerd
   become: yes
   apt_key:
     url: https://download.docker.com/linux/ubuntu/gpg
```
Comment (marianoleonardo) [inline roles/k8s-cluster/all-nodes/tasks/main.yaml:21]: This url is working for containerd?

Comment (danielapetito) [reply roles/k8s-cluster/all-nodes/tasks/main.yaml:21]: Yes.. The containerd is download from oficial repository of the docker as recommend of site Kubernetes.
https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd

image

**Review by marianoleonardo — APPROVED** [review]
LGTM!

**Review by gabrielifg — APPROVED** [review]
Nice!!!

**Pull request merged**